packages/ui/src/components/Checkbox/index.tsx

```diff
@@ -218,6 +277,7 @@ export const Checkbox = forwardRef(
       onBlur,
       error,
       name,
+      helper,
       value,
       size = 24,
```

Comment:
Can we change the default size to 20 so it follow the design specifications? Current checkbox is too big

![Screenshot 2023-08-23 at 14 44 47](https://github.com/scaleway/ultraviolet/assets/15812968/c3308b0b-2638-42d5-8e6b-711c9bdf56ac)

Reply:
My bad the sizes are correct